backend/infrahub/workflows/utils.py

```diff
@@ -18,7 +18,11 @@
 
 
 async def add_tags(
-    branches: list[str] | None = None, nodes: list[str] | None = None, others: list[str] | None = None
+    branches: list[str] | None = None,
+    nodes: list[str] | None = None,
+    others: list[str] | None = None,
+    namespace: bool = True,
+    db_change: bool = False,
```

Comment:
I don't know how much more we want to add to this but I think we might be getting to the point where we should instead have a single dataclass that we pass in as an argument to `add_tags` there we could also use the `field` to provide a description of the params.

Reply:
it should be it but agreed if we need more in the future we might need to revisit that part